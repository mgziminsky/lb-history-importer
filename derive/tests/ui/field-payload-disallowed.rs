use lb_importer_derive::IntoPayload;

#[derive(IntoPayload)]
struct PayloadFieldError {
    #[payload]
    track: String,
    other: String,
}

fn main() {}
