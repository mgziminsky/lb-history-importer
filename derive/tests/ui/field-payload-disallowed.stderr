error: Not allowed on fields. Must be placed on outer struct
 --> tests/ui/field-payload-disallowed.rs:5:5
  |
5 |     #[payload]
  |     ^^^^^^^^^^
